reproject/interpolation/core_celestial.py

```diff
@@ -6,8 +6,10 @@
 from ..wcs_utils import convert_world_coordinates
 from ..array_utils import iterate_over_celestial_slices, map_coordinates
 
+from astropy.utils.console import ProgressBar
 
-def _reproject_celestial(array, wcs_in, wcs_out, shape_out, order=1):
+def _reproject_celestial(array, wcs_in, wcs_out, shape_out, order=1, array_out=None,
+                         return_footprint=True, progressbar=True):
```

Comment:
The progress bar should be off by default. Also, can you rename ``progressbar`` to ``progress_bar``?